client-react/src/dashboard/CategoryBox.js

```diff
@@ -102,93 +138,72 @@ const CategoryBox = ({ title }) => {
-        </TitleWrapper>
-        <ContentWrapper>
-          <SubcategoryGrid>
-            <SubHeader>
-              <span>SUB-CATEGORIES</span>
-            </SubHeader>
-            <SubHeader>
-              <div>BEGINNER</div>
-              <div>EXPERTISE</div>
-            </SubHeader>
-            <div></div>
-          </SubcategoryGrid>
-          <Divider />
-          {subcategoryRender}
-        </ContentWrapper>
-        <FooterWrapper>
-          <SubHeader>{count} / 5 </SubHeader>
-          <FontAwesomeIcon
-            icon={faPlusCircle}
-            color="#2EC4B6"
-            size="2x"
-            style={{ marginLeft: "10px" }}
-            onClick={() => setCount(count < 5 ? count + 1 : 5)}
-          />
-        </FooterWrapper>
-      </CategoryWrapper>
-    );
+  const timeoutAnimation = () => {
+    if (display) {
+      setAnimate(false);
+      setDisplay(false);
+    } else {
+      setDisplay(true);
+      setAnimate(true);
+    }
   };
 
-  const hideCategoryContent = () => {
-    return (
-      <CategoryWrapper>
-        <TitleWrapper>
-          <div style={{ display: "flex", alignItems: "center" }}>
-            <span>
-              <FontAwesomeIcon
-                icon={faCaretRight}
-                size="lg"
-                onClick={() => setDisplay(!display)}
-              />
-            </span>
-            <Title>{title}</Title>
+  return (
+    <CollapseWrapper>
+      <EllipsisWrapper>
+        <FontAwesomeIcon
+          icon={faEllipsisH}
+          color="#707070"
+          onClick={() => setDropdownDisplay(!dropdownDisplay)}
+        />
+        {dropdownDisplay && (
+          <div ref={ref}>
+            <StyledDropdown />
           </div>
-          <EllipsisWrapper ref={ref}>
-            <FontAwesomeIcon
-              icon={faEllipsisH}
-              color="#707070"
-              onClick={() => setDropdownDisplay(!dropdownDisplay)}
-            />
-            {dropdownDisplay && (
-              <div ref={ref}>
-                <StyledDropdown />
-              </div>
-            )}
-          </EllipsisWrapper>
-        </TitleWrapper>
-      </CategoryWrapper>
-    );
-  };
-
-  return display ? showCategoryContent() : hideCategoryContent();
+        )}
+      </EllipsisWrapper>
+      <StyledCollapse defaultActiveKey="1" onChange={timeoutAnimation}>
+        <Panel
+          header={title}
+          style={{
+            fontFamily: "Quicksand",
+            fontSize: "15pt",
+            backgroundColor: "white",
+            padding: "15px",
+          }}
+        >
+          {display && (
+            <>
+              <ContentWrapper animate={animate}>
+                <SubcategoryGrid>
+                  <SubHeader>
+                    <span>SUB-CATEGORIES</span>
+                  </SubHeader>
+                  <SubHeader>
+                    <div>BEGINNER</div>
+                    <div>EXPERTISE</div>
+                  </SubHeader>
+                  <div></div>
```

Comment:
If you remove this empty div from the HTML dom, there's no visual change. Is there a reason why this is needed?